src/components/Export/Export.jsx

```diff
@@ -1,14 +1,21 @@
 import React, { useState } from "react";
-import { useSelector } from "react-redux";
+// import { useSelector } from "react-redux";
 import { Button, TextInput } from "carbon-components-react";
 import { Save16 } from "@carbon/icons-react";
-import { saveJsonToDisk, stateMappingToShifterMapping } from "./utils";
+import { saveAs } from "file-saver";
+import styles from "./export.module.scss";
 
-import styles from "./from_stix.module.scss";
-
-const Export = () => {
+const Export = (props) => {
```

Comment:
please add proptypes for type checking https://reactjs.org/docs/typechecking-with-proptypes.html